src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -121,4 +124,34 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
         }
         return tagSet;
     }
+
+    /**
+     * Parses a String slack into a Slack.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given slack is invalid.
+     */
+    public static Slack parseSlack(String slack) throws ParseException {
+        requireNonNull(slack);
+        String trimmedSlack = slack.trim();
+        if (!Slack.isValidSlack(slack)) {
+            throw new ParseException(Slack.MESSAGE_CONSTRAINTS);
+        }
+        return new Slack(slack);
+    }
+
+    /**
+     * Parses a String slack into a Slack.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given slack is invalid.
+     */
+    public static Telegram parseTelegram(String tele) throws ParseException {
+        requireNonNull(tele);
```

Comment:
`tele` is a bit confusing, probably a full name  `telegram` is better and consistent across the codebase.